docs/overrides/partials/comments.html

```diff
@@ -0,0 +1,59 @@
+{% if page.meta.comments %}
+
+{% if config.site_url and 'keptn.sh' in config.site_url %}
```

Comment:
wouldn't be more safe to have 
```suggestion
{% if config.site_url and 'https://keptn.sh/' in config.site_url %}
```
just to avoid registering `http://keptn.shush.com/` and post on our repo :p